.github/workflows/cd.yml

```diff
@@ -18,11 +18,23 @@ jobs:
       with:
         python-version: '3.x'
 
+    - name: Get tag and version
+      id: util
+      run: |
+        echo "::set-output name=tag::${GITHUB_REF##*/}"
+        echo "::set-output name=version::$(echo $GITHUB_REF | sed -e s/'refs\/tags\/[a-zA-Z]*'/''/)"
```

Comment:
Grab the tag, and version.

Version will parse out any characters before the version numbers, to make it compatible with PyPi.